types.go

```diff
@@ -151,9 +151,9 @@ func (aed AER) ENU() ENU {
 	)
 
 	return ENU{
-		East:  aed.Range * Meters(sinTheta*cosPhi),
-		North: aed.Range * Meters(sinTheta*sinPhi),
-		Up:    aed.Range * Meters(cosTheta),
+		East:  aed.Range * Meters(sinPhi*cosTheta),
+		North: aed.Range * Meters(cosPhi*sinTheta),
+		Up:    aed.Range * Meters(sinTheta),
```

Comment:
This currently breaks the round trip from ENU/AER. The test suite is failing -- it may be worth double-checking this changeset.

```
go test -v ./...
=== RUN   TestWGS84LLAtoXYZ
--- PASS: TestWGS84LLAtoXYZ (0.00s)
=== RUN   TestWGS84ConversionCycle
--- PASS: TestWGS84ConversionCycle (0.00s)
=== RUN   TestWGS84LLAtoENU
--- PASS: TestWGS84LLAtoENU (0.00s)
=== RUN   TestENUAERRoundTrip
    wgs_test.go:103: 
        	Error Trace:	wgs_test.go:103
        	Error:      	Relative error is too high: 1e-06 (expected)
        	            	        < 1.6832815729997475 (actual)
        	Test:       	TestENUAERRoundTrip
    wgs_test.go:104: 
        	Error Trace:	wgs_test.go:104
        	Error:      	Relative error is too high: 1e-06 (expected)
        	            	        < 0.49999999999999983 (actual)
        	Test:       	TestENUAERRoundTrip
    wgs_test.go:105: 
        	Error Trace:	wgs_test.go:105
        	Error:      	Relative error is too high: 1e-06 (expected)
        	            	        < 0.25464400750006994 (actual)
        	Test:       	TestENUAERRoundTrip
--- FAIL: TestENUAERRoundTrip (0.00s)
=== RUN   TestWGS84AERENURoundTrip
    wgs_test.go:132: 
        	Error Trace:	wgs_test.go:132
        	Error:      	Relative error is too high: 1e-07 (expected)
        	            	        < 0.00031912353329071854 (actual)
        	Test:       	TestWGS84AERENURoundTrip
    wgs_test.go:133: 
        	Error Trace:	wgs_test.go:133
        	Error:      	Relative error is too high: 1e-07 (expected)
        	            	        < 0.00024273941836575185 (actual)
        	Test:       	TestWGS84AERENURoundTrip
    wgs_test.go:134: 
        	Error Trace:	wgs_test.go:134
        	Error:      	Relative error is too high: 1e-07 (expected)
        	            	        < 113.64958114763722 (actual)
        	Test:       	TestWGS84AERENURoundTrip
--- FAIL: TestWGS84AERENURoundTrip (0.00s)
FAIL
FAIL	pault.ag/go/geo	0.002s
FAIL
```